src/main/kotlin/org/jitsi/nlj/srtp/SrtpUtil.kt

```diff
@@ -141,6 +141,10 @@ class SrtpUtil {
                 srtpProfileInformation.cipherSaltLength
             )
 
+            /* To support RetransmissionSender.retransmitPlain, we need to allow send-side SRTP replay. */
+            /* TODO: enable this only in cases where we actually need to use retransmitPlain? */
+            srtpPolicy.isSendReplayEnabled = true
```

Comment:
`sendReplayEnabled` controls "[[w]hether send-side replay **protection** is enabled](https://github.com/jitsi/jitsi-srtp/blob/master/src/main/java/org/jitsi/srtp/SrtpPolicy.java#L98)" (and defaults to true), so if we want `retransmitPlain` to work we need to **disable** the protection.